### DISCUSSION TITLE
Pedestrain About

### DISCUSSION THREAD

**Discussion by ADASDevelopment** [carla-simulator/carla#7732] [General]
Pedestrains don't work for me. (Unreal Engine/Carla on the development screen) When I do generate_[traffic.py](javascript:void(0);), walkers do not appear. How did you do it in RoadRunner? Or where do I need to fix it?

**Comment by wwzz2020bobo** [comment]
I'm having a similar issue where pedestrians won't generate on the map I imported from RoadRunner, even though I followed the instructions to create pedestrian navigation. Have you found a solution to this, or could you share any advice?

**Comment by loopsmark** [comment]
Did you generate the [pedestrian navigation](https://carla.readthedocs.io/en/latest/tuto_M_generate_pedestrian_navigation/#generate-pedestrian-navigation)? Without the pedestrian navigation that script won't find any suitable spawn points for the pedestrians.
